geoipupdate/test.py

```diff
@@ -0,0 +1,58 @@
+import shutil
```

Comment:
The tests file name and path should be `geoipupdate/tests/test_geoip_update.py`, also your test didn't ran on travis.

Reply:
I believe changing the path and name will fix the issue on travis.